prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -654,7 +654,7 @@ void rxr_pkt_handle_send_error(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entr
 			 */
 			rxr_cq_write_tx_error(ep, pkt_entry->x_entry, FI_ENORX, 0);
 			rxr_pkt_entry_release_tx(ep, pkt_entry);
-		} if (prov_errno == IBV_WC_RNR_RETRY_EXC_ERR) {
+		} else if (prov_errno == IBV_WC_RNR_RETRY_EXC_ERR) {
```

Comment:
I think the code would be easier to understand that `prov_errno == IBV_WC_RNR_RETRY_EXC_ERR` has its own block.

Reply:
Thanks. I updated in the new rev.